<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LxMemoClass</class>
 <widget class="QWidget" name="LxMemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item row="2" column="0">
    <widget class="QToolButton" name="button_back">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="LxMemo.qrc">
       <normaloff>:/LxMemo/icons/back.png</normaloff>:/LxMemo/icons/back.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="NoteWall" name="note_wall" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="TitleBar" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget{background-color: rgb(85, 170, 127);}
</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 0px;
border: 0px;
background-color: rgb(255,251,240);
margin-left: 4px;</string>
       </property>
       <property name="placeholderText">
        <string>搜索...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton{
	background-color: rgb(255,251,240);
	border:0px groove gray;
	border-radius:0px;
	padding:0px 4px;
	border-style: outset;
	margin-right: 4px;
}
QToolButton:hover{
	background-color:rgb(229, 241, 251); 
	color: black;}
QToolButton:pressed{
	background-color:rgb(204, 228, 247);
	border-style: inset;}</string>
       </property>
       <property name="text">
        <string>搜索</string>
       </property>
       <property name="icon">
        <iconset resource="LxMemo.qrc">
         <normaloff>:/LxMemo/icons/search.png</normaloff>:/LxMemo/icons/search.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header>TitleBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NoteWall</class>
   <extends>QWidget</extends>
   <header>notewall.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="LxMemo.qrc"/>
 </resources>
 <connections/>
</ui>
